### PR TITLE
JDK-8301466: [AIX] Revisit CommittedVirtualMemoryTest

### PR DESCRIPTION
**Pull request by varada1110**
We can now enable AIX on [CommitedVirtualMemoryTest](https://github.com/openjdk/jdk/blob/master/test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp#L35)  

JBS Issue : [JDK-8301466](https://bugs.openjdk.org/browse/JDK-8301466)

---------
### Progress
- [x] Change must be properly reviewed (1 review required, with at least 1 [Reviewer](https://openjdk.org/bylaws#reviewer))
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue

### Issue
 * [JDK-8301466](https://bugs.openjdk.org/browse/JDK-8301466): [AIX] Revisit CommittedVirtualMemoryTest (**Bug** - P4)

### Reviewers
 * [Martin Doerr](https://openjdk.org/census#mdoerr) (@TheRealMDoerr - **Reviewer**)

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jdk.git pull/17371/head:pull/17371` \
`$ git checkout pull/17371`

Update a local copy of the PR: \
`$ git checkout pull/17371` \
`$ git pull https://git.openjdk.org/jdk.git pull/17371/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 17371`

View PR using the GUI difftool: \
`$ git pr show -t 17371`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jdk/pull/17371.diff">https://git.openjdk.org/jdk/pull/17371.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jdk/pull/17371#issuecomment-1887050062)

### CODE REVIEW

**Comment by TheRealMDoerr** [comment]
This seems to break other platforms. And it doesn't enable anything on AIX.

**Comment by TheRealMDoerr** [comment]
I guess you should do a complete backout of https://hg.openjdk.org/jdk/jdk/rev/e1b3def12624?

**Comment by varada1110** [comment]
> I guess you should do a complete backout of https://hg.openjdk.org/jdk/jdk/rev/e1b3def12624?

It has already removed from here [src/hotspot/share/nmt/memTracker.hpp](https://github.com/openjdk/jdk/blob/master/src/hotspot/share/nmt/memTracker.hpp#L196) . All the tests under jdk/test/hotspot/gtest are passing.

**Comment by varada1110** [comment]
> This seems to break other platforms. Or is anything wrong with the GitHub Actions?

I noticed the same on other PRs as well.

**Comment by RealCLanger** [comment]
> > This seems to break other platforms. Or is anything wrong with the GitHub Actions?
> 
> I noticed the same on other PRs as well.

Yes, there's an issue currently with GHA. Some wget fails when assembling jtreg on all platforms. I guess GHA would need to be retriggered after that is resolved.

**Comment by RealCLanger** [comment]
> > I guess you should do a complete backout of https://hg.openjdk.org/jdk/jdk/rev/e1b3def12624?
> 
> It has already removed from here [src/hotspot/share/nmt/memTracker.hpp](https://github.com/openjdk/jdk/blob/master/src/hotspot/share/nmt/memTracker.hpp#L196) . All the tests under jdk/test/hotspot/gtest are passing.

Yes. That was done by https://github.com/openjdk/jdk/commit/201cdc50f9ec99fc1ee82236715119715fd2fdb9

File: test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp [hunk 1]
```diff
@@ -32,7 +32,6 @@
 class CommittedVirtualMemoryTest {
 public:
   static void test() {
-#ifndef _AIX
     // See JDK-8202772: temporarily disabled.
```
[the remark this answers is not in the document]
Comment (varada1110) [reply test/hotspot/gtest/runtime/test_committed_virtualmemory.cpp]: I have removed the comment
Thank you

**Review by TheRealMDoerr — APPROVED** [review]
Ok. Looks good, now. We should test it on our machines, too.

**Comment by varada1110** [comment]
Thank you
/integrate

**Comment by RealCLanger** [comment]
> Thank you /integrate

Hi Varada, could you please trigger GitHub Actions once again before we sponsor this? I'll also run it through SAP's testing. Thanks.

**Comment by TheRealMDoerr** [comment]
SAP's tests have passed. Let's ship it!
/sponsor

**Pull request closed without merging**